server/chat.ts

```diff
@@ -1368,6 +1368,8 @@ export const Chat = new class {
 		});
 	}
 	translationsLoaded = false;
+	readonly MAX_TIMEOUT_DURATION = 2147483647; // (2^32 / 2) - 1
```

Comment:
You may want to provide a link to Node.js documentation explaining the purpose of a variable: https://nodejs.org/api/timers.html#timers_settimeout_callback_delay_args

Reply:
Sure.